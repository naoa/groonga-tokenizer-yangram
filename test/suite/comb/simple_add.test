register tokenizers/yangram

yangram_register 2 --filter_combhira 1 --filter_combkata 1
tokenize TokenYaBigramFilterCombhiraCombkata "今日は雨だ" NormalizerAuto --mode ADD
