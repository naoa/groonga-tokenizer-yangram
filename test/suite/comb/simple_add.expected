register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register 2 --filter_combhira 1 --filter_combkata 1
[[0,0.0,0.0],"TokenYaBigramFilterCombhiraCombkata"]
tokenize TokenYaBigramFilterCombhiraCombkata "今日は雨だ" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0
    },
    {
      "value": "日は",
      "position": 1
    },
    {
      "value": "雨だ",
      "position": 2
    },
    {
      "value": "だ",
      "position": 3
    }
  ]
]
